lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -90,10 +96,19 @@ def edit_subscriber(self, subscriber_id):
             subscriber_data.ParseFromString(row[0])
             yield subscriber_data
             data_str = subscriber_data.SerializeToString()
-            self.conn.execute(
-                "UPDATE subscriberdb SET data = ? " "WHERE subscriber_id = ?",
-                (data_str, subscriber_id),
-            )
+            try:
```

Comment:
If we're going to do retries, let's put this in a loop w/ some max retry limit -- until we actually fix the DB lock problem then it's not clear to me that two retries is going to cut it.

Reply:
The second attempt is really about dithering (i.e., to break synchronization in a burst of requests). Since UE will timeout on its end if we do more backoffs, it has diminishing benefit (or no real benefit).